packages/andrjs/src/api/ADOSchemaAPI.ts

```diff
@@ -1,42 +1,26 @@
 import { Schema } from "jsonschema";
 import ADOAPI from "./ADOAPI";
 import AndromedaClient, { ContractSchema } from "index";
+import axios from "axios";
 
-export default class ADODSchemaAPI extends ADOAPI {
-  constructor(client: AndromedaClient, public address: string) {
+export default class ADOSchemaAPI extends ADOAPI {
+  constructor(client: AndromedaClient, public address: string = "") {
     super(client, address);
   }
 
-  /**
-   * Provides a message object for the Primitive DB's `GetKeyValue` query
-   * @param key
-   * @returns
-   */
-  getValueQuery(key: string) {
-    return {
-      get_value: {
-        key: key,
-      },
-    };
-  }
-
   /**
    * Gets the schema for the key in schemadb
    * @param key
    * @param address
    * @returns
    */
-  async getSchemaFromKey<T = ContractSchema>(key: string, address?: string) {
-    const msg = this.getValueQuery(key);
-    const res = await this.client.queryContract<{
-      key: string;
-      value: {
-        string: string;
-      }
-    }>(address ?? this.address, msg);
-    const schema = JSON.parse(res.value.string) as T;
+  private async getSchemaFromVersion<T = ContractSchema>(adoVersion: string, subSchema = 'default') {
+    const [adoType, version] = adoVersion.split('@');
+    const subPath = subSchema === 'default' ? adoType : subSchema;
+    const schema: T = await axios.get(`https://andromeda-schema-api.vercel.app/api/raw/${adoType}/${version}/${subPath}`).then(res => res.data);
```

Comment:
This is fine for a test path, but it should be swapped out for the official  api route before publishing changes.